SportHub.API/MapperConfig.cs

```diff
@@ -49,6 +49,13 @@ public MapperConfig()
 
             CreateMap<TeamCreateDto, Team>()
                 .ForMember(dest => dest.TeamId,opt => opt.MapFrom(src => Guid.NewGuid()));
+            
+            CreateMap<MainArticle, MainArticleRequest>();
+            CreateMap<MainArticleRequest, MainArticle>()
+                .ForMember(dest => dest.MainArticleId, opt => opt.MapFrom(src => Guid.NewGuid()))
```

Comment:
Guid.NewGuid is part of business logic - move creation of new guid when you create main article